frontend/src/exams/capture-ind-ita-base.vue

```diff
@@ -0,0 +1,507 @@
+<template>
+  <div>
+    <b-form>
+      <template v-if="step===1">
```

Comment:
What is step `1`? Can you use an enum here so it's more obvious what each step is?

Reply:
this is no longer relevant as i have completely reconsidered the process of rendering the question templates in response to the previous issue.